css/basscss.css

```diff
@@ -162,58 +137,170 @@ pre {
   overflow-x: scroll;
 }
 
+h1 { font-size: 2rem }
+h2 { font-size: 1.5rem }
+h3 { font-size: 1.25rem }
+h4 { font-size: 1rem }
+h5 { font-size: .875rem }
+h6 { font-size: .75rem }
+
+body {
+  color: #111;
+  background-color: white;
```

Comment:
I do this too so it isn't a criticism. But out of curiosity why the switch from hex to string in the same ruleset?